system/jlib/jtrace.cpp

```diff
@@ -770,7 +1011,8 @@ class CTraceManager : implements ITraceManager, public CInterface
         }
 
         //Administrator can choose to process spans in batches or one at a time
-        std::unique_ptr<opentelemetry::v1::sdk::trace::SpanProcessor> processor;
+        //Default: SimpleSpanProcesser sends spans one by one to an exporter.
+        std::unique_ptr<opentelemetry::v1::sdk::trace::SpanProcessor> processor = opentelemetry::sdk::trace::SimpleSpanProcessorFactory::Create(std::move(exporter));;
```

Comment:
This is a bug fix we need regardless, right now our default otel tracing setup does not define a default processor, which we didn't notice since we are ignoring the processor/exporter output